Content/default/webpack.config.js

```diff
@@ -80,7 +86,7 @@ module.exports = {
     plugins: isProduction ?
         commonPlugins.concat([
             new MiniCssExtractPlugin({ filename: 'style.[name].[hash].css' }),
-            new CopyWebpackPlugin({ patterns: [{ from: resolve(CONFIG.assetsDir) }]}),
+            new CopyWebpackPlugin({ patterns: [{ from: resolve(CONFIG.assetsDir) }] }),
         ])
         : commonPlugins.concat([
             new webpack.HotModuleReplacementPlugin(),
```

Comment:
Not completely necessary but since we already have `hot: true` in the devServer this plugin will be added by default, and we could remove it (check the tip inside the docs https://webpack.js.org/configuration/dev-server/#devserverhot)

Reply:
Done